ui/components/app/edit-gas-fee-popover/network-status/status-slider/status-slider.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+const GradientColors = [
```

Comment:
nit: Maybe this should be `gradientColors` or `GRADIENT_COLORS`?

Reply:
Fixed 👍